boards/renesas/da1469x_dk_pro/Kconfig.defconfig

```diff
@@ -5,8 +5,29 @@
 
 if BOARD_DA1469X_DK_PRO
 
+if LVGL
+
+# LCDC imposes display buffer be word aligned
+config LV_Z_VDB_ALIGN
+	default 4
```

Comment:
This should not be needed- the default value is already 4: https://github.com/zephyrproject-rtos/zephyr/blob/main/modules/lvgl/Kconfig.memory#L66

Reply:
Correct. The reason I also chose to add it here is because buffer alignment is critical for LCD controller's operation.